### DISCUSSION TITLE
It seems like recent MRs/PRs are adding TPM support?

### DISCUSSION THREAD

**Discussion by njfife** [Foxboron/sbctl#346] [General]
I apologize for my ignorance here, but I would like to keep TPM support for my dual boot system and and I'm trying to understand if it's possible for that to be done using sbctl at this point?

**Comment by Foxboron** [comment]
I don't understand the question.

What do you mean with "keep TPM support for my dual boot system"? What are you trying to do, and please be a bit more specific.
